WasatchNET/EEPROM.cs

```diff
@@ -1173,7 +1171,7 @@ public bool write()
                         logger.hexdump(pages[page], String.Format("writing page {0} [ARM]: ", page));
                         ok = spectrometer.sendCmd(
                             opcode: Opcodes.SECOND_TIER_COMMAND,
-                            wValue: (ushort)Opcodes.SET_MODEL_CONFIG_ARM,
+                            wValue: spectrometer.cmd[Opcodes.SET_MODEL_CONFIG_ARM],

```

Comment:
It blows my mind that this hadn't come up before now.  I was just trying to quickly patch an EEPROM field using WinFormDemo's EEPROMEditor, and was puzzled when it didn't work.  There's a huge difference between sending the enum value (SET_MODEL_CONFIG_ARM) and the opcode associated with that enum, so this never could have worked on ARM.

Reply:
considering we haven't built an ARM unit in Durham since I've been here I can't say I'm that surprised :), but good find